src/typegen.ts

```diff
@@ -317,13 +317,13 @@ export type GlobalMutationResolverParams = BaseMutationResolverParams & {
 }
 
 export type LocalMutationResolverParams<
-  MethodName extends MutationMethodName
+  MethodName extends any
```

Comment:
it extends `any` here because the rest of the types has generics extending any as well and I don't want to update the whole typegen code right now just to fix the initial problem.

Checks are made to make sure `MethodName` extends the available args anyway so nothing should go wrong